bin/iron/src/app.rs

```diff
@@ -3,24 +3,19 @@ use std::path::PathBuf;
 use iron_broadcast::UIMsg;
 use iron_db::DB;
 use iron_types::ui_events;
-#[cfg(not(target_os = "linux"))]
-use tauri::{AboutMetadata, Menu, MenuItem, Submenu, WindowMenuEvent};
 use tauri::{
-    AppHandle, Builder, CustomMenuItem, GlobalWindowEvent, Manager, SystemTray, SystemTrayEvent,
-    SystemTrayMenu, SystemTrayMenuItem, WindowBuilder, WindowEvent, WindowUrl,
+    AppHandle, Builder, GlobalWindowEvent, Manager, WindowBuilder, WindowEvent, WindowUrl,
 };
 use tauri_plugin_window_state::{AppHandleExt, Builder as windowStatePlugin, StateFlags};
 
-use crate::{commands, error::AppResult};
+use crate::{commands, error::AppResult, menu, system_tray};
```

Comment:
Ok, this fails beacuse you dont' have `system_tray` in macos: https://github.com/iron-wallet/iron/pull/351/files#diff-19ee4c60c2c483915400021b6e5b2e6d03f82e230f1b0c3112a44d33812fe5d9R8

Reply:
updated. can you try again?

I was trying to build for macos myself, but apparently can't be done on linux